src/main/java/homework_7/kittenToCatFunction/KittenToCatFunction.java

```diff
@@ -0,0 +1,6 @@
+package homework_7.kittenToCatFunction;
+
+@FunctionalInterface
+public interface KittenToCatFunction{
```

Comment:
Hi. Interface must accept only Cat, Kitten and their extends. Use generics.

Reply:
I am not shure about it. I asked this question on Stack and people answered that it would be meaningless at this very case.